batch/vep/run_script_with_watchdog.sh

```diff
@@ -23,18 +23,13 @@
 set -euo pipefail
 
 #################################################
-# Returns the generation number of a GCS file. In case of failure, it sleeps and
-# retries for at most 3 times.
+# Returns the generation number of a GCS file. Use the wrapped gsutil in
```

Comment:
I would put this comment at the top where you define the base image.

`The pipelines-io container provides a wrapper around gsutil with additional retry logic.`

Reply:
Done.